bin/runbrick.sh

```diff
@@ -41,18 +44,18 @@ echo -e "\nStarting on ${NERSC_HOST} $(hostname)\n" >> $log
 echo "-----------------------------------------------------------------------------------------" >> $log
 
 python -u legacypipe/runbrick.py \
-    --depth-cut \
     --max-blobsize  250000 \
-    --cache-dir /global/cscratch1/sd/dstn/dr5-new-sky/cache/ \
+    --normalize-psf \
+    --gaia \
+    --skip-calibs \
     --checkpoint $outdir/checkpoints/${bri}/checkpoint-${brick}.pickle \
```

Comment:
Once #196 is merged, `--normalize-psf` and `--gaia` will be set by default, so these should be removed here.  Should `--skip-calibs` also be a default, or perhaps it is already?

Finally, where does the `--max-blobsize` of 250000 come from?  Perhaps this value (if determined experimentally) should also go in #196.